docs/openapi.yaml

```diff
@@ -2354,8 +2349,7 @@ components:
           type: string
           description: |-
             The date when the charge is due to be processed.
-
-            Must be in the format `yyyy-MM-dd`
+            Must be in the format `YYYY-MM-DD`.
```

Comment:
we do NOT want to use week-based year (YYYY vs yyyy) and day-of-year (DD vs dd) AFAIK

https://dev.to/shane/yyyy-vs-yyyy-the-day-the-java-date-formatter-hurt-my-brain-4527

Reply:
Internal RFC link: https://github.com/vippsas/vipps-rfc/blob/master/docs/0001-api-guidelines.md#parameters

Reply:
Ah, yes, just specify that it is ISO 8601, then it'll be great :D